l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
@sadamo the problem is in the line 127 there is another definition of the attribute _order https://github.com/kmee/l10n-brazil/blob/916ad3cf2e80d1e755972034ddc8943f4d34c061/l10n_br_account/models/account_invoice.py#L127

You can remove the line 127 and change _order atribute to: _order = "date_invoice desc, internal_number desc" and in l10n_br_account_product chante to _order = "date_hour_invoice desc, internal_number desc"